tmt/utils/__init__.py

```diff
@@ -2446,29 +2461,104 @@ def from_env(cls) -> 'TracebackVerbosity':
 
 
 def render_run_exception_streams(
-        stdout: Optional[str],
-        stderr: Optional[str],
-        verbose: int = 0) -> Iterator[str]:
+        output: CommandOutput,
+        verbose: int = 0,
+        comment_sign: str = '#') -> Iterator[str]:
     """ Render run exception output streams for printing """
 
-    for name, output in (('stdout', stdout), ('stderr', stderr)):
-        if not output:
+    for name, content in (('stdout', output.stdout), ('stderr', output.stderr)):
+        if not content:
             continue
-        output_lines = output.strip().split('\n')
+        content_lines = content.strip().split('\n')
         # Show all lines in verbose mode, limit to maximum otherwise
         if verbose > 0:
-            line_summary = f"{len(output_lines)}"
+            line_summary = f"{len(content_lines)}"
         else:
-            line_summary = f"{min(len(output_lines), OUTPUT_LINES)}/{len(output_lines)}"
-            output_lines = output_lines[-OUTPUT_LINES:]
+            line_summary = f"{min(len(content_lines), OUTPUT_LINES)}/{len(content_lines)}"
+            content_lines = content_lines[-OUTPUT_LINES:]
 
-        yield f'{name} ({line_summary} lines)'
-        yield OUTPUT_WIDTH * '~'
-        yield from output_lines
-        yield OUTPUT_WIDTH * '~'
+        yield f'{comment_sign} {name} ({line_summary} lines)'
+        yield comment_sign + (OUTPUT_WIDTH - 1) * '~'
```

Comment:
```suggestion
        yield f'{comment_sign} ' + (OUTPUT_WIDTH - 2) * '~'
```

Would you be ok with this as well? I find it a little bit more pleasing to the eye...

Reply:
I don't mind, just need to fix tests after such a change.

Reply:
That would be nice, thanks!

Reply:
Addressed in 21f18965

Reply:
Thanks!